soql-analyzer/src/main/scala/com/socrata/soql/SoQLAnalyzer.scala

```diff
@@ -350,8 +361,10 @@ class SoQLAnalyzer[Type](typeInfo: TypeInfo[Type],
   private def aliasContext(tableName: Option[TableName], ctx: AnalysisContext): AnalysisContext = {
     tableName match {
       case Some(TableName(TableName.SingleRow, alias)) =>
+        alias.foreach(validateAlias(_))
```

Comment:
Hahah oh god someone could have done `FROM @single_row AS this` and I hate that and thank you for nipping that before it budded.